package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bs-dom-testing-library",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
Revert this as it is done when published to npm.

Reply:
Done.